NOTES.UNIX

```diff
@@ -89,3 +90,22 @@
 
     $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
         '-Wl,--enable-new-dtags,-rpath,$(LIBRPATH)'
+
+ It might be worth noting that some/most ELF systems implement support
+ for relative runtime search path, relative to directory containing
+ the executable object, by interpreting $ORIGIN along with some other
+ internal variables. Consult your system documentation.
+
+ Linking your application
+ ------------------------
+
+ Above is the story of building and installing OpenSSL shared libraries
+ and application. But third-party applications dynamically linked with
```

Comment:
As mentioned in #6560, I don't think the first sentence is necessary at all (yes, I can see such a construct it a piece of literature, to create a sense of continuity between pieces of a story, but I don't think that applies here).  The whole paragraph can start with "Third party applications ..."